guides/05_custom-components/08_multimodal-chatbot-part1.md

```diff
@@ -0,0 +1,355 @@
+# Build a Custom Multimodal Chatbot - Part 1
+
+This is the first in a two part series where we build a custom Multimodal Chatbot component.
+In part 1, we will modify the Gradio Chatbot component to display text and media files (video, audio, image) in the same message.
+In part 2, we will build a custom Textbox component that will be able to send multimodal messages (text and media files) to the chatbot.
+
+You can follow along with the author of this post as he implements the chatbot component in the following YouTube video!
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/IVJkOHTBPn0?si=bs-sBv43X-RVA8ly" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
Looks like the video needs to be made public: 
<img width="604" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/776516dd-2039-4174-bf57-7d9650dfa1b8">

Reply:
Yep! It's going to automatically publish in 10 minutes!